src/mgmt/srv_chk.c

```diff
@@ -122,6 +122,32 @@ ds_mgmt_check_act(uint64_t seq, uint32_t act)
 	return chk_leader_act(seq, act);
 }
 
+int
+ds_mgmt_check_set_policy(uint32_t policy_nr, Mgmt__CheckInconsistPolicy **policies)
+{
+	struct chk_policy *ply = NULL;
+	int                rc;
+	int                i;
+
+	if (policy_nr == 0)
+		return -DER_INVAL;
```

Comment:
Would be good to log a `D_ERROR` message in this case.

Reply:
sure

Reply:
sure